mmedit/models/losses/feature_loss.py

```diff
@@ -0,0 +1,94 @@
+import torch
+import torch.nn as nn
+from mmcv.runner import load_checkpoint
+
+from mmedit.models.components.discriminators import LightCNN
+from mmedit.utils import get_root_logger
+from ..registry import LOSSES
+
+
+class LightCNNFeature(nn.Module):
+    """Feature of LightCNN.
+
+    It is used to train DICGAN.
+    """
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        model = LightCNN(3)
+        self.features = nn.Sequential(*list(model.features.children()))
+        for _, v in self.features.named_parameters():
+            v.requires_grad = False
+
+    def forward(self, x):
+        """Forward function.
+
+        Args:
+            x (Tensor): Input tensor.
+
+        Returns:
+            Tensor: Forward results.
+        """
+
+        return self.features(x)
+
+    def init_weights(self, pretrained=None, strict=True):
```

Comment:
Will you call this function?

Reply:
Yes, the model should be pre-trained in training and testing.